### PR TITLE
Fix age cannot be float number issue

### LINKED ISSUE #152: age_numerical cannot allow float numbers
Doesn't alllow accept >1.5(18 months).

### PR DESCRIPTION
**Pull request by victoria34**
Fix #152

### CODE REVIEW

File: src/main/webapp/app/clinical/clinical.component.ts [hunk 1]
```diff
@@ -48,13 +48,12 @@ export class ClinicalComponent implements OnInit {
             const ageGroups = this.clinicalInput.age_numerical.split(',');
             // Age input cannot only accepts age range groups greater than 2.
             if (ageGroups.length === 2) {
-                const ageNumber0 = Number(ageGroups[0].match(/\d\d?$/));
-                const ageNumber1 = Number(ageGroups[1].match(/\d\d?$/));
+                const ageNumber = this.clinicalInput.age_numerical.match(/\d+(\.?\d+)?/g).map(function(v) { return parseFloat(v); });
```
Comment (zhx828) [inline src/main/webapp/app/clinical/clinical.component.ts]: Why don't we just use lodash _.isNumber?

Comment (victoria34) [reply src/main/webapp/app/clinical/clinical.component.ts]: _.isNumber() returns boolean value. I need to convert string to float number.

Comment (zhx828) [reply src/main/webapp/app/clinical/clinical.component.ts]: You just need to know whether `clinicalInput.age_numerical` is number then you can just converted a string a number by doing Number(string)

File: src/main/webapp/app/clinical/clinical.component.ts [hunk 2]
```diff
@@ -87,11 +86,13 @@ export class ClinicalComponent implements OnInit {
         }
     }
     validateSingleAgeStr(age_numerical: string) {
-        if (age_numerical.match(/^(>|>=|<|<=)\d\d?$/)) {
```
Comment (zhx828) [inline src/main/webapp/app/clinical/clinical.component.ts]: /^((>|<)?=?\d\d?$/

**Pull request merged**